docs/getting_started/tasks_and_workflows.md

```diff
@@ -130,7 +130,7 @@ def standard_scale_workflow(values: List[float]) -> List[float]:
 Just like tasks, workflows are executable in a regular Python runtime:
 
 ```{code-cell} ipython3
-standard_scale_workflow(values=[float(i) for i in range(1, 11)])
+print(standard_scale_workflow(values=[float(i) for i in range(1, 11)]))
```

Comment:
`print` isn't required as the code is enclosed in a jupyter notebook cell which automatically prints the output of the command.